eppoclient/client_test.go

```diff
@@ -290,3 +291,39 @@ func Test_client_correctActionIsReturnedIfBanditLoggerPanics(t *testing.T) {
 		Action:    &expectedAction,
 	}, result)
 }
+
+func Test_Initialized_timeout(t *testing.T) {
+	var mockLogger = new(mockLogger)
+	client := newEppoClient(newConfigurationStore(), nil, nil, mockLogger, applicationLogger)
+
+	timedOut := false
+	select {
+	case <-client.Initialized():
+		timedOut = false
+	case <-time.After(1 * time.Millisecond):
+		timedOut = true
```

Comment:
According to ChatGPT, this select block will wait for one of these two channels to complete, similar to JavaScript's Promise.race() 👌